src/data.rs

```diff
@@ -287,6 +304,7 @@ mod printing {
                 Visibility::Restricted(ref path) => {
                     tokens.append("pub");
                     tokens.append("(");
+                    tokens.append("in");
```

Comment:
Right, but that would also affect `test_round_trip`, just in the reverse direction. If the input contains `in super` the output would contain `super`. Though there isn't such a file right now, so I guess it's fine.